apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-overflow-example.component.html

```diff
@@ -43,18 +43,18 @@
         style="max-width: 200px;"
     />
 
-    <button fd-toolbar-item fd-button label="Button" [fdMenu]="true" [compact]="true"></button>
+    <button fd-toolbar-item fd-button fdToolbarOverflowButton fdToolbarOverflowButtonMenu label="Button" [fdMenu]="true" [compact]="true"></button>
```

Comment:
Don't you find it weird to have 2 different naming conventions on the same element? `fd-button fdToolbarOverflowButton`????
Please use only one.

Reply:
Updated example.